enterprise/internal/a8n/resolvers/resolver.go

```diff
@@ -362,8 +362,9 @@ func (r *Resolver) Campaigns(ctx context.Context, args *graphqlbackend.ListCampa
 	if err := allowReadAccess(ctx); err != nil {
 		return nil, err
 	}
-	opts := ee.ListCampaignsOpts{
-		State: a8n.CampaignState(args.State),
+	var opts ee.ListCampaignsOpts
+	if args.State != nil {
+		opts.State = a8n.CampaignState(*args.State)
```

Comment:
Hmm, do we need to validate that the string->a8n.CampaignState conversion worked?

```suggestion
		s := a8n.CampaignState(*args.State)
		if !s.Valid() {
		  return nil, errors.New("Cannot filter invalid Campaign state")
		}
		opts.State = s
```

Reply:
Done